File: Front/FFStats-App/src/app/Model/Partida.ts
import { Call } from "./Call";
import { Estatistica } from "./Estatistica";
import { Jogador } from "./Jogador";
import { Mapa } from "./Mapa";
import { Modo } from "./Modo";
import { Submodo } from "./Submodo";
import { Treino } from "./Treino";

export interface Partida {

  id: number;
  partidaDescricao: string;
  treinoId: number;
  treino: Treino;
  mapaId: number;
  mapa: Mapa;
  callId: number;
  call: Call;
  modoId: number;
  modo: Modo;
  submodoId: number;
  sumodo: Submodo;
  partidaData?: Date;
  posicao: number;
  partidasJogadores: Jogador[];
  estatisticas: Estatistica[];
  MVP: string

}

File: Back/src/FFStats.Domain/Models/Partida.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace FFStats.Domain.Models
{
    [Table("tb_partida")]
    public class Partida
    {
        public int Id { get; set; }
        public string partidaDescricao { get; set; }
        public int treinoId { get; set; }   
        public Treino treino { get; set; }
        public int mapaId { get; set; }
        public Mapa mapa { get; set; }
        public int callId { get; set; }
        public Call call { get; set; }
        public int modoId { get; set; }
        public Modo modo { get; set; }
        public int submodoId { get; set; }
        public Submodo sumodo { get; set; }
        public DateTime? partidaData { get; set; }
        public int posicao { get; set; }
        public IEnumerable<PartidaJogador> PartidasJogadores { get; set; }
        public IEnumerable<Estatisticas> Estatisticas { get; set; }
    }
}